### PR TITLE
UIREC-53 Update "receive" piece Modal to full screen

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
Update "receive" piece Modal to full screen
https://issues.folio.org/browse/UIREC-53

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/TitleDetails/TitleDetails.js [hunk 1]
```diff
@@ -107,25 +105,28 @@ const TitleDetails = ({
...
 
-  const onReceivePiece = useCallback(
-    (piece) => {
-      onReceive(piece);
-      toggleReceivingModal();
-    },
-    [onReceive, toggleReceivingModal],
+  const expectedPiecesActions = useMemo(
+    () => (
+      <>
```
Comment (NikitaSedyx) [inline src/TitleDetails/TitleDetails.js]: please create a component for this

File: src/TitleReceive/TitleReceive.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+} from '@folio/stripes-acq-components';
+
+import {
+  PIECE_FORMAT_LABELS,
+} from '../common/constants';
+
+const visibleColumns = ['checked', 'barcode', 'format', 'hasRequest', 'comments', 'location', 'itemStatus', 'callNumber'];
+const field = 'receivedItems';
+
+const TitleReceiveList = ({ fields, props: { locationOptions, selectLocation, toggleCheckedAll } }) => {
```
Comment (NikitaSedyx) [inline src/TitleReceive/TitleReceive.js]: 2 components in one file?

File: src/TitleReceive/TitleReceiveContainer.js [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+  useEffect(
+    () => {
+      if (poLineId) {
+        mutator.pieces.GET({
+          params: {
+            query: `poLineId==${poLineId} and receivingStatus==${PIECE_STATUS.expected} sortby locationId`,
+          },
+        })
+          .then((piecesResponse) => {
+            const itemsIds = piecesResponse.filter(({ itemId }) => itemId).map(({ itemId }) => itemId);
```
Comment (NikitaSedyx) [inline src/TitleReceive/TitleReceiveContainer.js]: !

File: src/TitleReceive/TitleReceive.js (hunk 2 above)
Comment (aliaksei-chumakou) [reply src/TitleReceive/TitleReceive.js]: yeah, but they are coupled a lot

File: src/TitleReceive/TitleReceiveContainer.js (hunk 3 above)
Comment (aliaksei-chumakou) [reply src/TitleReceive/TitleReceiveContainer.js]: logic is leveraged into a fn `getHydratedPieces`

File: src/TitleDetails/TitleDetails.js (hunk 1 above)
Comment (aliaksei-chumakou) [reply src/TitleDetails/TitleDetails.js]: new component is created

**Pull request merged**